src/solvers/refinement/string_constraint_generator_format.cpp

```diff
@@ -365,7 +366,7 @@ add_axioms_for_format_specifier(
                     << message.eom;
     INVARIANT(
       false, "format specifier must belong to [bBhHsScCdoxXeEfgGaAtT%n]");
-    throw 0;
+    throw unreachable_code_exceptiont("Unknown format specifier.");
```

Comment:
That seems an overkill. I'd be fine with `throw 0` in this case.

Reply:
or `UNREACHABLE;` should be fine now